migrations/main.sql

```diff
@@ -633,3 +633,27 @@ CREATE VIEW user_request_booking_hostel_view AS
     ORDER BY
         u.university_id,
         s.user_id;
+
+
+DROP VIEW IF EXISTS return_user_document_view;
+CREATE VIEW return_user_document_view AS
+    SELECT
+        ur.university_id,
+        ur.date_created,
+        ud.name,
+        ud.user_document_id,
+        ur.user_id
+    FROM
+        user_request ur
+    LEFT JOIN status st ON
```

Comment:
why did u add this join?
i thnk u forgot to add filtering of the status of user requests. If so, please add it